code/controllers/subsystem/vote.dm

```diff
@@ -164,14 +164,14 @@ SUBSYSTEM_DEF(vote)
 				if(. == "Restart Round")
 					restart = 1
 			if("gamemode")
-				if(master_mode != .)
+				if(GLOB.master_mode != .)
 					world.save_mode(.)
 					if(SSticker && SSticker.mode)
 						restart = 1
 					else
-						master_mode = .
-				if(!going)
-					going = 1
```

Comment:
oh, that's what "going" stood for. why was that file labeled "unused". why would anyone include something that is not used.